src/proto/flwr/proto/transport.proto

```diff
@@ -119,3 +122,27 @@ message Scalar {
     bytes bytes = 15;
   }
 }
+
+message Message {
+  Metadata metadata = 1;
+  RecordSet content = 2;
+  Error error = 3;
+}
+
+message Context {
+  sint64 node_id = 1;
+  map<string, Scalar> node_config = 2;
+  RecordSet state = 3;
+  map<string, Scalar> run_config = 4;
+}
+
+message Metadata {
+  sint64 run_id = 1;
+  string message_id = 2;
+  sint64 src_node_id = 3;
+  sint64 dst_node_id = 4;
+  string reply_to_message = 5;
+  string group_id = 6;
+  double ttl = 7;
+  string message_type = 8;
+}
```

Comment:
Let's move these into a new `.proto` file. `transport.proto` covers mostly the legacy gRPC stack and we don't want to add new functionality to it. How about just `message.proto`?

Reply:
Good point, I'll update it.